tools/ci_build/github/azure-pipelines/templates/win-ci.yml

```diff
@@ -15,6 +15,16 @@ parameters:
   CudaVersion: ''
   AgentPool: 'Win-CPU'
   AgentDemands: []
+  trigger:
+    branches:
+      include:
+      - master
```

Comment:
We need to include the "rel-*" branches as well or else this won't get triggered in the release branches.